packages/storybook/src/components/form/form-group-checkboxes.css-template.ts

```diff
@@ -30,8 +30,9 @@ export const cssFormGroupCheckboxes: ComponentImplementation<FormGroupCheckboxes
           aria-describedby=${ifDefined(ariaDescribedBy)}
           aria-errormessage=${ifDefined(ariaErrorMessage)}
         >
+          <legend class="sr-only">${formGroup.label}</legend>
           <div class="dso-label-container">
-            <label for=${formGroup.id} class="control-label"> ${formGroup.label} </label>
+            <span class="control-label" aria-hidden="true"> ${formGroup.label} </span>
```

Comment:
Als je hier dna toch bent, wil je die spaties weghalen? Dit is nog en overblijfsel van Prettier